filter.txt

```diff
@@ -1242,4 +1244,4 @@ zoomtech.ir###text-3
 ||zarpop.com^$script
 ||zoomit.ir/*.gif
 ||zoomit.ir/*/declarationPics/*.jpg
-||zoomtech.ir/wp-content/uploads/*.gif
\ No newline at end of file
+||zoomtech.ir/wp-content/uploads/*.gif
```

Comment:
Same as above. Unless the URL is invalid or it breaks the page rendering.

Reply:
I actually did not change this line, the diff is generated by GitHub when there is no blank newline at the end of the file.